### PR TITLE
Leeds Chapter

### PR DESCRIPTION
**Pull request by notshi**
Hi @angiemaguire and @iteles

Hope this pr looks ok for the Leeds Chapter.

The repo is also a ```gh-pages``` website - [currently looks like this](https://notshi.github.io/leeds/).

If someone with admin access can enable the settings below, it would be very much appreciated!

Settings > Options > Github Pages > Source > **master branch /docs folder**

Thank you!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +0,0 @@
-<img align="right" width="250px" src="https://cloud.githubusercontent.com/assets/8995723/12703902/58324a9e-c846-11e5-9f8f-49326881efaf.gif"/>
```
Comment (iteles) [inline README.md:1]: Even though we aren't actively using the _main_ the readme, can we please aim to keep it in?
It shouldn't cause a conflict with gh-pages and a repo without a readme is a little confusing for most people - always good to let people know what's going on and how they can contribute!

File: LICENSE [hunk 2]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2016 Ladies of Code
+Copyright (c) 2017 Ladies of Code
```
Comment (iteles) [inline LICENSE]: :tada:

**Review by iteles — APPROVED** [review]
👍 This is awesome, thanks for this @notshi!!

I left one small comment.

I don't seem to have the option to enable the docs branch 😕 
Leaving this to @angiemaguire as I'm not sure whether it's a permissions thing or not (I usually deploy from `master` and leave `docs` to documentation so haven't come across this!).
screen shot 2017-10-09 at 20 11 48

File: README.md (hunk 1 above)
Comment (notshi) [reply README.md:1]: Thanks, @iteles!

I did include an updated [readme.md](https://github.com/notshi/leeds/blob/f96af5d7257342eba5ce1bbc829fd26fb0b01ceb/readme.md) with this pr albeit lowercase renaming.

Sorry, I may have misunderstood what you meant!

Comment (iteles) [reply README.md:1]: No, you're absolutely right, I must have missed it! 🤦🏻‍♀️

Looks great!

Comment (notshi) [reply README.md:1]: Yay! *phew* Thank you :}

**Comment by notshi** [comment]
Hi @angiemaguire and @iteles,

Just a short note about enabling the docs branch.

The branch is only available once the pull request is merged - more info [here](https://help.github.com/articles/configuring-a-publishing-source-for-github-pages/#publishing-your-github-pages-site-from-a-docs-folder-on-your-master-branch).

Hope that helps! Thanks!

**Comment by notshi** [comment]
Hello @angiemaguire and @iteles!

I've updated the code base with our latest event, as well as, new additions to our speaker list and keeping to the format of the London chapter (deleted the extra gubbins).

Hope this pull request gets merged! Would be great to be able to point new members and potential speakers to this page so they have an idea of our past meetings and plans for future ones.

Many thanks in advance!

~ shi

**Comment by iteles** [comment]
@notshi Merging this based on @angiemaguire's 👍's! Taking that as approval from her side too 😉 

Thanks again for keeping this up to date!

**Comment by notshi** [comment]
Thank you, @iteles!

**Pull request merged**